src/Inkluzitron/Modules/Points/PointsModule.cs

```diff
@@ -35,5 +54,98 @@ public async Task GetPointsAsync(IUser member = null)
 
             await ReplyFileAsync(points.Path);
         }
+
+        [Command("board")]
+        [Alias("list")]
+        [Summary("Žebříček uživatelů s nejvíce body.")]
+        public async Task GetLeaderboardAsync()
+        {
+            await GetLeaderboardAsync(0);
+        }
+
+        [Command("board")]
+        [Alias("list")]
+        [Summary("Žebříček uživatelů s nejvíce body s posunem od počátku tabulky.")]
+        public async Task GetLeaderboardAsync([Name("offset")]int start)
+        {
+            var count = await PointsService.GetUserCount();
+
+            if (start >= count) start = count - 1;
+            start -= start % BoardPageLimit;
+
+            var board = PointsService.GetLeaderboard(start, BoardPageLimit);
+
+            var embed = new PointsEmbed().WithBoard(
+                board, Client, Context.Client.CurrentUser, count, start, BoardPageLimit);
+
+            var message = await ReplyAsync(embed: embed.Build());
+            await message.AddReactionsAsync(ReactionSettings.PaginationReactions);
+        }
+
+        [Command("board")]
+        [Alias("list")]
+        [Summary("Žebříček uživatelů s nejvíce body. Zobrazí žebříček kolem zadaného uživatele.")]
+        public async Task GetLeaderboardAsync([Name("uživatel")] IUser user)
+        {
+            var pos = await PointsService.GetUserPosition(user);
+            if (pos < 0)
+                await ReplyAsync("Tento uživatel nemá žádný záznam o bodech.");
+            else
+                await GetLeaderboardAsync(pos);
+        }
+
+        public async Task<bool> HandleReactionAddedAsync(IUserMessage message, IEmote reaction, IUser user)
```

Comment:
Ten reaction handler bych vytáhnul do samostatné třídy. Mám trochu obavu, když ta jedna třída bude registrovaná 2x, tak aby to nedělalo v DC.NET chaos.

Reply:
Už je to tak nějakou dobu i v Urban. Takže to mám předělat i tam?